contracts/interface/IPriceFeedDispatcher.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity 0.7.6;
 
+import "./IPriceFeed.sol";
```

Comment:
here

Reply:
Fixed in [f45027b](https://github.com/perpetual-protocol/perp-oracle-contract/pull/85/commits/f45027b21ff698d37962fdaf316340e596569b4c)